ui/app/AppLayouts/Chat/stores/MessageStore.qml

```diff
@@ -12,6 +12,13 @@ QtObject {
     readonly property int newMessagesCount: messagesModel ? messagesModel.newMessagesCount : 0
     readonly property bool messageSearchOngoing: messageModule ? messageModule.messageSearchOngoing : false
 
+    readonly property bool amIChatAdmin: messageModule ? messageModule.amIChatAdmin : false
+    readonly property bool isPinMessageAllowedForMembers: messageModule ? messageModule.isPinMessageAllowedForMembers : false
+    readonly property string chatId: messageModule ? messageModule.getChatId() : ""
+    readonly property int chatType: messageModule ? messageModule.chatType : 0
```

Comment:
would it make sense to return `Constants.chatType.unknown` instead?

Reply:
Totally, will update